src/app/share-buttons/share-buttons.component.scss

```diff
@@ -5,6 +5,11 @@
     top: 200px;
     padding: 0px;
 
+    // Avoid blocking important content on mobile
```

Comment:
Does this hide the social share buttons on mobile? Is the PO okay with that?

Reply:
I gave Steve a full demo, including the full website in mobile, but I am not sure if this was discussed specifically. The way it was setup before, it was actively blocking all of the content in front of it. So if we re-integrated it, it would need to at least stop doing that...

Reply:
I talked to him about it this morning and he said it's fine to leave out but we'll need to create a ticket to replace it. He said he'll be doing that shortly